driver/registry_default_registration.go

```diff
@@ -6,12 +6,7 @@ import (
 )
 
 func (m *RegistryDefault) PostRegistrationHooks(credentialsType identity.CredentialsType) []registration.PostHookExecutor {
-	a := m.hooksPost(credentialsType, m.c.SelfServiceRegistrationAfterHooks(string(credentialsType)))
-	b := make([]registration.PostHookExecutor, len(a))
-	for k, v := range a {
-		b[k] = v
-	}
-	return b
+	return m.hooksPostRegistration(credentialsType, m.c.SelfServiceRegistrationAfterHooks(string(credentialsType)))
```

Comment:
Do we even need separate `hooksPostLogin/Registration`? I would rather inline it here.